dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestLineInterface.cpp

```diff
@@ -178,9 +173,17 @@ TEST(DataInterfaceTest, Line) {
                                        .setG2(2.0)
                                        .setB2(0.4)
                                        .add();
-  LineInterfaceIIDM li2(MyBadLine);
+  MySecondLine.getTerminal1().disconnect();
+  MySecondLine.getTerminal2().disconnect();
 
+  LineInterfaceIIDM li2(MySecondLine);
+  ASSERT_FALSE(li2.getInitialConnected1());
+  ASSERT_FALSE(li2.getInitialConnected2());
   ASSERT_EQ(li2.getR(), 0.0);
   ASSERT_EQ(li2.getX(), 0.01);
+  ASSERT_DOUBLE_EQ(li2.getP1(), 0.0);
+  ASSERT_DOUBLE_EQ(li2.getQ1(), 0.0);
```

Comment:
the set of P are missing (we also get 0 is P is nan so we do need to set them to a value)

Reply:
I add 2 cases for P/Q: one with "inf" another one with "nan"

Reply:
done